src/main/java/Class/Event.java

```diff
@@ -0,0 +1,15 @@
+package Class;
+
+public class Event extends Task {
+    protected String at;
```

Comment:
This variable name could refer to a location or time aspect of event. You could consider creating a name that could give [more explicit meaning](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain).